pkg/app/metadata.json

```diff
@@ -0,0 +1,963 @@
+      },
+      "list": {
+        "apis": ["https://developer.fastly.com/reference/api/logging/syslog/#list-log-syslog"]
+      },
+      "update": {
+        "apis": ["https://developer.fastly.com/reference/api/logging/syslog/#update-log-syslog"]
+      }
+    }
+  },
+  "pops": {
+    "apis": ["https://developer.fastly.com/reference/api/utils/pops/#list-pops"]
+  },
+  "purge": {
+    "apis": [
+      "https://developer.fastly.com/reference/api/purging/#purge-all",
+      "https://developer.fastly.com/reference/api/purging/#bulk-purge-tag",
+      "https://developer.fastly.com/reference/api/purging/#purge-tag",
+      "https://developer.fastly.com/reference/api/purging/#purge-single-url"
+    ]
+  },
+  "search": {
+    "create": {
+      "apis": ["https://developer.fastly.com/reference/api/services/service/#create-service"]
+    },
+    "delete": {
+      "apis": [
+        "https://developer.fastly.com/reference/api/services/service/#get-service-detail",
+        "https://developer.fastly.com/reference/api/services/version/#deactivate-service-version",
+        "https://developer.fastly.com/reference/api/services/service/#delete-service"
+      ]
+    },
+    "describe": {
+      "apis": ["https://developer.fastly.com/reference/api/services/service/#get-service-detail"]
+    },
+    "list": {
+      "apis": ["https://developer.fastly.com/reference/api/services/service/#list-services"]
+    },
+    "search": {
+      "apis": ["https://developer.fastly.com/reference/api/services/service/#search-service"]
+    },
+    "update": {
+      "apis": ["https://developer.fastly.com/reference/api/services/service/#update-service"]
+    }
+  },
+  "service-version": {
+    "activate": {
+      "apis": ["https://developer.fastly.com/reference/api/services/version/#activate-service-version"]
+    },
+    "clone": {
+      "apis": ["https://developer.fastly.com/reference/api/services/version/#clone-service-version"]
+    },
+    "deactivate": {
+      "apis": ["https://developer.fastly.com/reference/api/services/version/#deactivate-service-version"]
+    },
+    "list": {
+      "apis": ["https://developer.fastly.com/reference/api/services/version/#list-service-versions"]
+    },
+    "lock": {
+      "apis": ["https://developer.fastly.com/reference/api/services/version/#lock-service-version"]
+    },
+    "update": {
+      "apis": ["https://developer.fastly.com/reference/api/services/version/#update-service-version"]
+    }
+  },
+  "stats": {
+    "historical": {
+      "apis": ["https://developer.fastly.com/reference/api/metrics-stats/historical-stats/#get-hist-stats-service"]
+    },
+    "realtime": {
+      "apis": ["https://developer.fastly.com/reference/api/metrics-stats/realtime/#get-stats-last-second"]
+    },
+    "regional": {
+      "apis": ["https://developer.fastly.com/reference/api/metrics-stats/historical-stats/#get-regions"]
+    }
+  },
+  "update": {
```

Comment:
I presume I should just remove commands like this, where there are neither any APIs used nor an examples currently implemented.